src/scripts/email-screenshots.sh

```diff
@@ -13,15 +13,14 @@ m_pwd="yoursender_password"
 m_file="/tmp/imgmail.html"
 m_data="/tmp/imgmail.txt"
 
-# Save the backlight to a variable, set it to 100, then set it back after imgur.sh ?
 bklght=$(lv GUI_backlightUserRequest)
-sdv GUI_backlightUserRequest 100
+sdv GUI_backlightUserRequest 255 && sleep 1
 CID=$(curl -s http://cid:4070/screenshot)
-sleep 2
 IC=$(curl -s http://ic:4130/screenshot)
-sleep 2
+sdv GUI_backlightUserRequest ${bklght//\"}
```

Comment:
Why was the `${bklght//\"}` necessary? Did just `$bklght` not work?

Reply:
Somehow it didn't. Brightness always went up to 255 (100%) with the plain variable. Stripping quotes from it works, although quotes are no problem when passing a quoted value directly. Couldn't figure out why it behaves differently when the value comes from a variable.